pvlib/pvsystem.py

```diff
@@ -944,63 +945,61 @@ def physicaliam(aoi, n=1.526, K=4., L=0.002):
     return iam
 
 
-def calcparams_desoto(poa_global, temp_cell, alpha_isc, module_parameters,
-                      EgRef, dEgdT, M=1, irrad_ref=1000, temp_ref=25):
+def calcparams_desoto(effective_irradiance, temp_cell,
+                      alpha_sc, a_ref, I_L_ref, I_o_ref, R_sh_ref, R_s, 
```

Comment:
I will [repeat](https://github.com/pvlib/pvlib-python/issues/469#issuecomment-394407197) my preference for spelling out most of these arguments in the function signature as in [i_from_v](http://pvlib-python.readthedocs.io/en/latest/generated/pvlib.pvsystem.i_from_v.html#pvlib.pvsystem.i_from_v). It would require adding some mapping function within ``PVSystem.calcparams_desoto`` but I think that's worth it for more descriptive argument names. The descriptive parameter names could be reassigned to the short names to keep the equations more readable.

The abbreviated names are still an improvement to the existing API, so I do not object to moving forward as is after some consideration.

Reply:
Works for me.

Reply:
The R_s instead of R_s_ref in SAM has always puzzled me.